### PR TITLE
Update name parameter for DHCP service create

### PR DESCRIPTION
**Pull request by guruhosmath**
Update name parameter for DHCP service create

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #[6208](https://jsw.ibm.com/browse/POWERIAAS-6208)

Output from acceptance testing:
```
$ make testacc TESTARGS='-run=TestAccIBMPIDhcpWithDhcpServerName'
PASS: TestAccIBMPIDhcpWithDhcpServerName (428.55s)
PASS
```

### CODE REVIEW

File: website/docs/r/pi_dhcp.html.markdown [hunk 1]
```diff
@@ -49,6 +49,8 @@ Review the argument references that you can specify for your resource.
 
 - `pi_cloud_instance_id` - (Required, String) The GUID of the service instance associated with an account.
 - `pi_cloud_connection_id` - (Optional, String) The Cloud Connection UUID to connect with DHCP private network.
+- `pi_dhcp_server_name` - (Optional, String) Name of the dhcp server.
```
Comment (yussufsh) [inline website/docs/r/pi_dhcp.html.markdown:52]:
```suggestion
- `pi_dhcp_server_name` - (Optional, String) Name of the DHCP service.
```

File: website/docs/r/pi_dhcp.html.markdown [hunk 2]
```diff
@@ -49,6 +49,8 @@ Review the argument references that you can specify for your resource.
...
+- `pi_dhcp_dns_server` - (Optional, String) IP address of dns server for dhcp server.
```
Comment (yussufsh) [inline website/docs/r/pi_dhcp.html.markdown:53]:
```suggestion
- `pi_dhcp_dns_server` - (Optional, String) DNS Server for DHCP service.
```

File: go.mod [hunk 3]
```diff
@@ -57,3 +57,5 @@ require (
 replace github.com/softlayer/softlayer-go v1.0.3 => github.com/IBM-Cloud/softlayer-go v1.0.5-tf
 
 replace github.com/dgrijalva/jwt-go v3.2.0+incompatible => github.com/golang-jwt/jwt v3.2.1+incompatible
+
+replace github.com/IBM-Cloud/power-go-client => github.com/guruhosmath/power-go-client v1.0.84
```
Comment (yussufsh) [inline go.mod:61]: You will need latest commit hash or tag.

**Review by yussufsh — CHANGES_REQUESTED** [review]

**Comment by yussufsh** [comment]
@guruhosmath any updates?

**Comment by guruhosmath** [comment]
Changes went in as part of https://github.com/IBM-Cloud/terraform-provider-ibm/pull/3916 . So closing this PR.

**Pull request closed without merging**